### DISCUSSION TITLE
Sharing excel files with CoolProp formulas in it

### DISCUSSION THREAD

**Discussion by oci95** [CoolProp/CoolProp#2407] [Q&A]
Hello everybody, I have a problem.
In my organization we use the excel add-in of coolprop, excel files are shared in a private server.
When I save an excel with CololProp formulas in it in the server, and then the same file is opened by one of my colleague, all coolprop formulas has the the address of my PC (C:\Users\*USER*\AppData\Roaming\Microsoft\AddIns). I try to put the xlam file in the shared server and I changed the address of the file in the add-ins option. Unfortunatly, everytime that excel is run it automatically changes the address to the local one,
How is it possible to solve it?

**Comment by ibell** [comment]
In short, I think there is no easy solution. This is an architectural thing about how addins work. I would strongly recommend to migrate to a solution that does not require a shared add-in.

**Comment by henningjp** [comment]
This is a hazard of Excel add-und.

If you have a shared drive area and can install the .xla, .xlm code from there, then everyone will point to the same location for the functions and it won’t change.  This will take some experimentation on your part to load the add-in manually (after uninstalling the local copies).

The re is an added benefit in that these add-ins will all have to be code signed eventually with newer versions of Office365.  With a shared copy, on person can code sign the add-in for everyone in your community.

Alternatively, do a global replace on the leading local path name on all the functions and replace with an empty string, “”.  Then the macros will find your own local copy.  Not ideal, but you can do it in one go.
